packages/server/src/authentication/verifyAuthenticationResponse.ts

```diff
@@ -197,36 +204,35 @@ export async function verifyAuthenticationResponse(
     }
   }
 
-  const extensionOutputs: ExtensionOutputs = {};
+  const extensionOutputs: AuthenticationExtensionOutputs = {};
 
   if (flags.ed) {
-    if (!extensionsData && !clientExtensionResults) {
+    if (!clientExtensionResults) {
       throw new Error('Authenticator data indicated extension data was present,'+
-        ' but no client or authenticator extension data were found');
+        ' but no client extension data were found');
```

Comment:
I actually think this line should drop "client" for "authenticator" since my reading of https://www.w3.org/TR/webauthn-2/#authenticator-data tells me that only authenticator extensions are present in `authenticatorData`.
```suggestion
        ' but no authenticator extension data were found');
```